### PR TITLE
Replace bronze with gnucydur

### PR DESCRIPTION
**Pull request by FreeLikeGNU**
I argue that in MT, as steel (an alloy of iron and carbon) can be made from iron alone, then bronze (another alloy)  be made from copper alone.
Steel items should be an upgrade to bronze items and have nerfed bronze thus.  
Copper ingots and blocks have been removed as bronze replaces them.
Thank you for your consideration of this pull request.

### CODE REVIEW

**Comment by PilzAdam** [comment]
The idea is that copper is an upgrade to steel ingots, to increase their durability. Making bronze tools completely different from steel is useless, since stone tools are good enough to get to steel tools.

Also in the current mapgens, copper ores seems to be more rare than iron ore. Thus making steel better than bronze renders copper/bronze useless.

**Comment by FreeLikeGNU** [comment]
PilzAdam, I have also found that copper is a bit more rare than iron.   Regardless, it just hurts my brain that bronze could be fashioned from copper and steel.   So how about this: If we make bronze directly from copper and bronze items to be an upgrade to steel items, that we require bronze as the handle ingots (instead of wood)? Then the rarity of copper and upgrade rationale are preserved.

**Comment by PilzAdam** [comment]
@FreeLikeGNU needing wood for the handle forces players to go out of their caves sometimes, to get new wood from trees.

If it hurts your brain maybe just rename bronze to something else?

**Comment by FreeLikeGNU** [comment]
@PilzAdam another good point, also requiring wood encourages me to grow forests underground if not just for making torches.  How about renaming bronze to glucydur?
https://en.wikipedia.org/wiki/Glucydur

**Comment by CraigyDavi** [comment]
You need to add aliases from bronze to gnucydur too.

Personally I don't think the change is very user friendly. I've never heard of Glucydur and most people will have no idea what it is and where to get the material from.

**Comment by paramat** [comment]
Bronze in Minetest is controversial and perhaps a little messy, but i'm not too keen on this commit, especially introducing a bizarre named obscure metal. Any changes to bronze are better discussed carefully in an issue, but then this has been done before several times =)

**Comment by FreeLikeGNU** [comment]
I only offer some suggestions and PR solutions to mitigate the 'messiness' and 'controversy'.  I don't mind bronze in the game, but I think it should be fixed or replaced to make the game better.  If there are other ideas, I'm happy to help make it happen!  I realize minetest_game is to be the foundation for other subgames and mods, so lets clean it up!

**Comment by FreeLikeGNU** [comment]
@CraigyDavi I never heard of mese until minetest, I think it set the precedent for obscure materials in a game. ;-)

**Comment by C1ffisme** [comment]
Gnucydur isn't really heard of by many people. Sure, we want minetest to be different from minecraft in some ways, but I feel that the name is just a little overkill to make bronze be something different. What about using something that comes off the tounge easier? Like Iron Copite (IC)?

**Comment by FreeLikeGNU** [comment]
@C1ffisme I could not find Iron Copite in a web search, but that does not bother me at all.  Costeel, copiron, biron, brosteel may also be acceptable? Just not bronze - I think its just confusing to players and painful to anyone who knows anything about metals. :D

**Comment by CraigyDavi** [comment]
> I never heard of mese until minetest

Hrm, good point.

**Comment by FreeLikeGNU** [comment]
What about simply CuFe? 
http://www.fiskalloy.com/product/c194/

**Comment by paramat** [comment]
Apart from the crazy mese, Minetest materials are ment to be well known and related to the different 'ages' stone age, iron age, bronze age (though of course we have iron and bronze back to front). Minetest is very often highly unrealistic and simplistic in it's crafting. Let's stay with bronze, although i'm open to changes to bronze.
Personally i don't think bronze tools were a good addition as we already have wood stone iron/steel mese and diamond, that's plenty and bronze is too close to steel.
Bronze and copper ore is fine though.

**Comment by C1ffisme** [comment]
@FreeLikeGNU Iron Copite probably isn't a real material. I just came up with the chemical name for the molecule FeCo (Iron and Copper). Of course, there can't be only metal molecules, so you would have to put in something like carbon from coal, etc.

**Comment by FreeLikeGNU** [comment]
@paramat what changes to bronze would you be open to?

**Comment by Ferk** [comment]
Imho, for a game like this with much educational potential, it would be a pity for it to fall into misconceptions. It's ok to be simplistic, but there's no reason to feed the notion that iron is present in every single metal alloy. Bronze has no iron in its composition.

With so many metals for the tools already then there's little reason to have bronze be an "upgraded steel ingot". The upgrade from steel tools to bronze tools is very small anyway, just change bronze ingot to come directly from copper, as if it was a different version or iron, a simple variation from the steel tools, in a similar spirit at how there are variations for the wood.

**Comment by paramat** [comment]
FreeLikeGNU don't know, we'll deal with the suggestions when they appear.

Ferk i agree it's bad to teach that the sun is square.

**Comment by Ferk** [comment]
The sun not being being square is very obvious to anyone, and it's straightforward to understand that it's a simplification to adapt to the look and feel of the game.

The bronze recipe being unrealistic doesn't seem that obvious for someone who doesn't know what is bronze made of. I can imagine many people being misled. After all the rest of fundamental recipes in Minetest are pretty realistic (even if simplistic), like water+lava=obsidian, smelted sand=glass, smelted ores=lingot.. papyrus=paper .. wheat->flour->bread they all make sense in a way. Bronze is the only recipe so far in minetest_game that uses unrealistic materials.

**Comment by FreeLikeGNU** [comment]
@paramat so far what I have already suggested:
1. Bronze be made from copper alone with qualities between stone and steel - not acceptable
2. Gnucydur or some otherly named alternative name for copper/steel combo alloy - not acceptable
so heres another suggestion:
Bring tin back to combine with copper for bronze.  I personally dont think items made with bronze should be better than steel for RPG trope reasons, but at least its not painful to my narrow view of science and technology.

**Comment by Ferk** [comment]
Considering that steel can easily rust, a possible use of tin would be to replace steel for buckets.
The food mod could use it for canned food.
Or make tools more durable by combining steel+tin=stainless steel.

**Comment by FreeLikeGNU** [comment]
I agree that copper, tin and bronze could be useful in their own rights in minetest for the reasons @Ferk mentioned.  It would not bother me at all if these metals were useful for mainly non-weapon purposes and make them useful well past the time when player encounters iron, diamond and mese.

**Comment by paramat** [comment]
> Bring tin back to combine with copper for bronze

That's also unpopular with the devs.

-1 for this commit.
Perhaps this should be closed now?

**Comment by 0-afflatus** [comment]
Sure, but if you are going to have bronze the recipe should be copper + tin.
Bronze weapons and tools should be slower than iron, but more durable.
This is one of the reasons I started creating a sub-game. This PR doesn't add anything to Minetest Game.

**Comment by C1ffisme** [comment]
Actually about the name, I think gnucydur is actually fine. Why?

In Terraria, there is a material called Clorophyte, there is a boss called the Eye of Cuthulu, and there is a biome called the Corruption. I think it's good to introduce things people don't know about to make the game seem fuller of content. If you have some of the most wacky things like bar stools and 20 different types of cheese in your game, then you almost don't need to make all the normal things.

Basically, the game feels a lot bigger when there is stranger and more out-of-this-world items in the game. So even if your game has a normal or even small amount of content, just making it seem like the game has everything you could ever dream up can make it feel bigger.

**Comment by 0-afflatus** [comment]
I like Ferk's perspective on this one. Minetest does have an educational aspect. If you know that glass is made from sand in rl, you can guess at the crafting recipe. You should be able to make buckets out of tin, etc.. Otherworldly and fantastical elements should be provided by mods.

**Comment by C1ffisme** [comment]
Making buckets out of tin isn't that bad of an idea... Considering steel is already used in chest locks, steel blocks, and tools (I would use steel or bronze shovels and axes rather than use my diamonds on tools that get used a lot).

**Comment by Ekdohibs** [comment]
I think that though the iron + copper recipe is nonsensical, renaming it to something that no-one heard of is not a good idea. We should either keep the current recipe, or add tin and replace iron with tin.

**Comment by HybridDog** [comment]
>though the iron + copper recipe is nonsensical, renaming it to something that no-one heard of is not a good idea

in my opinion doing it the other way round would be a bad idea

**Comment by sofar** [comment]
I'm sad to see that people prefer adding "gnucydur" over "tin" and teach kids something about alloying.

Either
1. remove any resemblance to earth materials and make them all nonsensical or fantasy
or
2. add tin, and make steel require more coal during burning to simulate the adding of carbon into the metallurgic structure

**Comment by C1ffisme** [comment]
> make steel require more coal during burning 

What about making an alloy furnace? That would be more realistic, and means mod makers don't have to come up with enormous amounts of code to make realistic alloys and their recipes.

Also, if we really don't want bronze, why not alias all current bronze and copper items to iron/steel?

Or maybe just bronze items, and copper items could have some other use, like making buckets and locks in case you use all of your iron on tools. (A complaint of 0gb.us)

**Comment by sofar** [comment]
One could mix tin lumps with copper lumps to make 2 bronze lumps... that doesn't sound too far-fetched.

**Comment by C1ffisme** [comment]
Of course, sofar, if we were going for educational, bronze would be worse then steel.

Or, if we do that, we could just alias bronze *tools* to iron ones, and then keep bronze to make buckets / locks / item that does something useful in-game.

**Comment by sofar** [comment]
:-1:

**Comment by paramat** [comment]
-1 from sfan5 on IRC.

**Comment by C1ffisme** [comment]
You know, we could always just make copper a lot more common, and just turn current bronze tools into worse copper tools, which are just made from copper ingots.

**Comment by 0-afflatus** [comment]
@C1ffisme please let this thread die. Please.
